hydro_serving_writer/utils/io.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+def delete_file(path):
```

Comment:
why would you create this function when you can clearly replace it with `os.remove`?

Reply:
Sorry, forgot to remove that from index. I do not use it.